### PR TITLE
backport 936faba6c43 to 6.x

### PR DESCRIPTION
**Pull request by pcsanwald**
This PR backports delete job from `master` to `6.x` branch.

### CODE REVIEW

File: docs/java-rest/high-level/supported-apis.asciidoc [hunk 1]
```diff
@@ -300,14 +300,19 @@ include::migration/get-assistance.asciidoc[]
 :doc-tests-file: {doc-tests}/RollupDocumentationIT.java
 
 
+:upid: {mainid}-rollup
+:doc-tests-file: {doc-tests}/RollupDocumentationIT.java
```
Comment (cbuescher) [inline docs/java-rest/high-level/supported-apis.asciidoc]: duplication from line 299/300

**Review by cbuescher — CHANGES_REQUESTED** [review]
I left a few minor comments, looks good otherwise.

**Comment by pcsanwald** [comment]
@cbuescher no rush, but I believe I have addressed your comments and this is ready for re-review. thank you!

**Review by cbuescher — APPROVED** [review]
Many thanks, LGTM.

**Pull request merged**